tests/plugins/custommsg_a.py

```diff
@@ -5,9 +5,9 @@
 
 
 @plugin.hook('custommsg')
-def on_custommsg(peer_id, message, plugin, **kwargs):
+def on_custommsg(peer_id, payload, plugin, message=None, **kwargs):
     plugin.log("Got custommessage_a {msg} from peer {peer_id}".format(
-        msg=message,
+        msg=payload,
         peer_id=peer_id
```

Comment:
This should probably be simply rolled into the previous one, since otherwise there's bisect breakage.

It's pretty small...